app/views/recruitment_cycles/_locations_info.html.erb

```diff
@@ -1,8 +1,9 @@
 <h2 class="govuk-heading-m">
   <%= govuk_link_to "Locations", provider_recruitment_cycle_sites_path(provider.provider_code, year) %>
 </h2>
-<p class="govuk-body">Use this section to:</p>
+<p class="govuk-body">Use this section to add or edit your training locations. This will make your courses more visible on Find.</p>
+<p class="govuk-body">You can also choose whether to:</p>
 <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-8">
-  <li>edit location names and addresses</li>
-  <li>add locations</li>
+  <li>show precise placement locations or only the general area</li>
+  <li>allow candidates to choose a placement school</li>
 </ul>
```

Comment:
Keep this content as it was before (we are not implementing the features this copy relates to)

Reply:
Reverted all changes to this partial.